### PR TITLE
Sanitize the text field for good practice mutation

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
- Depends on #18 

## Changes
- Sanitize text field for good practice table
- Added a generic method for sanitizing text field except translated text field

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [ ] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] tests
- [x] permission checks (tests here too)
- [x] translations

### CODE REVIEW

File: apps/country/models.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from django.contrib.postgres.fields import ArrayField
 from django.utils.translation import gettext_lazy as _
 from utils import year_choices, current_year
+import config.custom_fields  # noqa F401
```
Comment (thenav56) [inline apps/country/models.py:5]: Let's add a NOTE as well on why we are doing this.

File: config/custom_fields.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import django
+from django.db.models import TextField
+from html_sanitizer import Sanitizer
+
+sanitizer = Sanitizer()
+
+
+def clean_text_field(field_value):
+    # TODO: find the way to override custom field in modeltranslation
```
Comment (thenav56) [inline config/custom_fields.py:9]: We can also do something like this
```python3
from modeltranslation.utils import build_localized_fieldname
from django.conf import settings


def clean_text_fields(obj, *fields):
    for field in fields:
        for lang, _ in settings.LANGUAGES:
            lang_field = build_localized_fieldname(field, lang)
            original_value = getattr(obj, lang_field, '')
            sanitized_value = sanitizer.sanitize(original_value)
            setattr(obj, lang_field, sanitized_value)
```

File: config/custom_fields.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+def get_db_prep_value(self, *args, **kwargs):
+    value = super(TextField, self).get_db_prep_value(*args, **kwargs)
+    if isinstance(value, str):
+        return sanitizer.sanitize(value)
+    return value
+
+
+django.db.models.TextField.get_db_prep_value = get_db_prep_value
```
Comment (thenav56) [inline config/custom_fields.py:21]: Let's remove this if not used to avoid confusion.

**Review by thenav56 — CHANGES_REQUESTED** [review]
Looks good. Few suggesstions.

File: config/custom_fields.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+sanitizer = Sanitizer()
+
+
+def clean_text_fields(obj, *fields):
+    for field in fields:
+        for lang, _ in settings.LANGUAGES:
+            lang_field = build_localized_fieldname(field, lang)
+            original_value = getattr(obj, lang_field, '')
+            print(original_value)
```
Comment (thenav56) [inline config/custom_fields.py:14]:
```suggestion
```

File: apps/good_practice/tests/test_mutations.py [hunk 5]
```diff
@@ -91,12 +91,32 @@ def test_create_goodpractice(self):
                 titleEn='some text title en',
                 implementingEntityEn='some text implementing entity en',
                 mediaAndResourceLinksEn='some text media and resource links en',
-                descriptionEn='some text description en',
+                # descriptionEn='some text description en',
```
Comment (thenav56) [inline apps/good_practice/tests/test_mutations.py:94]: Let's remove this

**Review by thenav56 — CHANGES_REQUESTED** [review]
LGTM. Few changes

**Review by bimal125 — APPROVED** [review]

**Review by thenav56 — APPROVED** [review]

**Pull request merged**